Bottom: 1653bd308112f591e2370c7679620e5e11f27ea0
Top:    16b22fc3f5f0219951d1d982ae9a6207ad149490
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-18 15:15:16 +0000

Refresh of fixup-move-of-step-stuff

---

diff --git a/src/hid/step/object3d_step.c b/src/hid/step/object3d_step.c
new file mode 100644
index 0000000..0d4db9d
--- /dev/null
+++ b/src/hid/step/object3d_step.c
@@ -0,0 +1,24 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <assert.h>
+#include <stdbool.h>
+
+#include <glib.h>
+
+#include "hid/common/step_id.h"
+#include "hid/common/quad.h"
+#include "hid/common/vertex3d.h"
+#include "hid/common/contour3d.h"
+#include "hid/common/appearance.h"
+#include "hid/common/face3d.h"
+#include "hid/common/edge3d.h"
+#include "hid/common/object3d.h"
+#include "polygon.h"
+#include "data.h"
+
+#include "step_writer.h"
+
+#include "pcb-printf.h"
+
+#include "object3d_step.h"
+
diff --git a/src/hid/step/object3d_step.h b/src/hid/step/object3d_step.h
new file mode 100644
index 0000000..5dc6f4d
--- /dev/null
+++ b/src/hid/step/object3d_step.h
@@ -0,0 +1,2 @@
+void object3d_list_export_to_step_assy (GList *objects, const char *filename);
+void object3d_export_to_step (object3d *object, const char *filename);
